erpnext/patches/v6_27/update_two_checkbox_value_for_recurringdoc.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+	for doctype in ('Sales Order', 'Purchase Order', 'Sales Invoice',
+		'Purchase Invoice'):
+		frappe.reload_doctype(doctype)
+		frappe.db.sql('''update `tab{0}` set submit_on_creation=0, notify_by_email=0
+			where is_recurring=1'''.format(doctype))
```

Comment:
will close this pull and update standard patch v6-24/set_recurring_id.py. In standard patch, set all submit_on_creation=0 where is_recurring=1 and don't need update notify_by_email because we introduce notify_by_email earlier than them and we already set value on it.